source/uwp/Renderer/lib/Util.cpp

```diff
@@ -805,33 +805,44 @@ std::string WstringToString(const std::wstring& input)
     return utfConverter.to_bytes(input);
 }
 
-void RemoteResourceElementToUriStringVector(
+void RemoteResourceElementToUriInformationVector(
```

Comment:
>Uri [](start = 28, length = 3)

Nit: Shouldn't it be RemoteResourceElementToRemoteResourceInfoVector or ToRemoteResourceInformationVector?